lib/sls-api-file-list-error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+class SlsApiFileListError extends Error {
+
+	static get messages() {
+
+		return {
+			BUCKET_NOT_DEFINED: 'Bucket not defined',
+			BUCKET_NOT_STRING: 'Bucket should be return a string'
```

Comment:
Sugerencia: Yo le pondria INVALID_BUCKET al nombre y algo como 'Bucket should be a string' al mensaje.